docs/config.md

```diff
@@ -45,6 +45,37 @@ with environment variables taking precedence over `App.config` or `Web.config` f
      `SiteName\VirtualPath` ex: `MySite\MyApp`
      - If that is not the case it will use the name of the application [entry Assembly](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getentryassembly?view=net-7.0).
 
+4. File-based Configuration (Experimental)
```

Comment:
What do you think about putting whole file based configuration documentation in separated file?
Maybe even as a follow up to this PR?